VSCode/armsnippets.json

```diff
@@ -506,6 +506,145 @@
     ],
     "description": "Availability Set"
 },
+"Azure Firewall": {
+    "prefix": "arm-firewall",
+    "body": [
+        "{",
+        "    \"name\": \"${1:firewall1}\",",
+        "    \"type\": \"Microsoft.Network/azureFirewalls\",",
+        "    \"apiVersion\": \"2019-04-01\",",
+        "    \"location\": \"[resourceGroup().location]\",",
+        "    \"tags\": {},",
+        "    \"properties\": {",
+        "       \"applicationRuleCollections\": [",
+        "           {",
+        "               \"name\": \"${2:appCollectionName1}\",",
+        "               \"properties\": {",
+        "                   \"priority\": \"${3|100,1-65000|}\",",
```

Comment:
I assume "1-65000" isn't a valid value, so I'd change this line to \"priority\": \"${3:100}\",",